client/src/test/java/se/fortnox/reactivewizard/client/HttpClientTest.java

```diff
@@ -139,6 +139,28 @@ public void shouldNotRetryFailedPostCalls() {
         }
     }
 
+    @Test
+    public void shouldNotRetryFailedPostCallsWithSystemExceptions() {
+        AtomicLong callCount = new AtomicLong();
+        HttpServer<ByteBuf, ByteBuf> server    = startServer(INTERNAL_SERVER_ERROR, "\"NOT OK\"", r -> callCount.incrementAndGet());
```

Comment:
perhaps add a delay and return 200 OK, to trigger the read timeout which is the error we are trying to fix?